templates/includes/card-group-record-summary-no-image.html

```diff
@@ -8,7 +8,14 @@
 <li class="col-sm-12 col-md-6 col-lg-4">
     <div class="card-group-record-summary">
         <a href="{% url 'details-page-machine-readable' iaid=record_page.iaid %}" class="card-group-record-summary__link" data-card-type="card-group-record-summary-no-image" data-card-title="{{ record_page.title }}">
-            <h3 class="card-group-record-summary__heading">{{ record_page.title }}</h3>
+            <p class="card-group-record-summary__heading">
+                {% if record_page.reference_number %}
+                <span class="sr-only">
+                    {{ record_page.reference_number }}
```

Comment:
Question, do we want the hyphen between the catalogue reference and title still for screen readers? e.g. `{{record_page.reference_number}} - ` otherwise the links  on  Dragon VoiceOver and the way it's read out by a screen reader may be different without the hyphen.